src/menu/menu.c

```diff
@@ -645,6 +717,9 @@ menu_process_cursor_motion(struct wlr_scene_node *node)
 	assert(node && node->data);
 	struct menuitem *item = node_menuitem_from_node(node);
 
+	if (!item || !item->selectable) {
+		return;
+	}
```

Comment:
IMHO the check for `!item` isn't necessary here because `node_menuitem_from_node()` already asserts if the type is wrong and the assert above asserts that there is in fact some kind of descriptor set. We could turn it into an additional `assert(item)` though.

Reply:
👍